scripts/read-config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# Yes, this is a Bad YAML Parser, but at this stage we are not in the
+# venv and do not know what modules the user has available, so for
+# maximum compatibility, we are just assuming a plain Python distribution.
+import argparse
+import re
+import sys
+
+parser = argparse.ArgumentParser()
+group = parser.add_mutually_exclusive_group(required=True)
+group.add_argument('--sources',action='store_true')
+group.add_argument('--family',action='store_true')
+args = parser.parse_args()
+
+with open("sources/config.yaml") as config:
```

Comment:
Nit. If we're expecting this to work on Win, use `os.path.join` instead of a forward slash.